git-extra/git-extra.install.in

```diff
@@ -122,6 +122,18 @@ test -d "$TMPDIR" || test ! -d "$TMP" || {\
 	grep -q ' exe "normal! g`\\""' ||
 	sed -i -e '/^ *autocmd BufReadPost \*/,/\\ endif$/s/^/"/' \
 		/usr/share/vim/vim80/defaults.vim
+		
+	line="$( grep -n "^# if we have the \"High Mandatory Level\" group, it means we're elevated\$" /etc/bash.bashrc )"
+	if [ -n "$line" ]; then
+		start="${line%%:*}"
+		if tail +$((start+1)) /etc/bash.bashrc | head -1 | grep -q '^if \[\[ -n "$(command -v getent)" \]\] .*'; then
+			line="$( tail +$((start+2)) /etc/bash.bashrc | grep -n '^fi$' )"
+		fi
+		if [ -n "$line" ]; then
+			end="$(($start+${line%%:*}+1))"
+			sed -ie "${start},${end}d" /etc/bash.bashrc
+		fi
+	fi
```

Comment:
Hmm. That looks a little bit complicated, and certainly a little hard to reason about. Why not just something like

```sh
! grep -q '^[^#].*command -v getent' /etc/bash.bashrc ||
sed -i '/command -v getent/,/^fi/s/^/#/' /etc/bash.bashrc
```

I.e. if a line is found that contains `command -v getent` and does *not* start with a `#`, then comment-out the lines starting with the one containing `command -v getent` and ending in the line starting with `fi`...